pilot/pkg/networking/core/v1alpha3/route/route_test.go

```diff
@@ -69,6 +69,14 @@ func TestBuildHTTPRoutes(t *testing.T) {
 		g.Expect(len(routes)).To(gomega.Equal(1))
 	})
 
+	t.Run("for virtual service with catch all route", func(t *testing.T) {
```

Comment:
This test case fails with out the code change and sends two routes because catch all match is at the end.